### PR TITLE
GlusterFS: Fix registry.yml playbook

### PR DESCRIPTION
**Pull request by jarrpa**
Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1590138

Signed-off-by: Jose A. Rivera <jarrpa@redhat.com>

### CODE REVIEW

**Comment by jarrpa** [comment]
@michaelgugino PTAL :)

/retest

**Comment by jarrpa** [comment]
@michaelgugino @mtnbikenc Looking for an LGTM.

File: playbooks/openshift-glusterfs/private/registry.yml [hunk 1]
```diff
@@ -5,9 +5,12 @@
...
   - role: openshift_persistent_volumes
-    when: openshift_hosted_registry_storage_glusterfs_swap | default(False)
+    when:
+    - openshift_hosted_registry_storage_kind | default(none) == 'glusterfs' or openshift_hosted_registry_storage_glusterfs_swap
+
+- import_playbook: ../../openshift_hosted/private/openshift_hosted_registry.yml
+
+- import_playbook: ../../openshift_hosted/private/openshift_hosted_wait_for_pods.yml
+
+- import_playbook: ../../openshift_hosted/private/openshift_hosted_registry_storage.yml
```
Comment (michaelgugino) [inline playbooks/openshift-glusterfs/private/registry.yml]: Looks like there is an errant blank line at the end of the file.  Please fix for lint tests.

File: playbooks/openshift-glusterfs/private/registry.yml [hunk 2]
```diff
@@ -5,9 +5,11 @@
...
   - role: openshift_persistent_volumes
-    when: openshift_hosted_registry_storage_glusterfs_swap | default(False)
+    when:
+    - openshift_hosted_registry_storage_kind | default(none) == 'glusterfs' or openshift_hosted_registry_storage_glusterfs_swap
 
-- name: Create Hosted Resources
-  hosts: oo_first_master
-  roles:
-  - role: openshift_hosted
+- import_playbook: ../../openshift_hosted/private/openshift_hosted_registry.yml
```
Comment (michaelgugino) [inline playbooks/openshift-glusterfs/private/registry.yml]: should be 'openshift-hosted' not 'openshift_hosted' in path name.

**Comment by jarrpa** [comment]
Changed again. :)

**Comment by michaelgugino** [comment]
bot, retest this please

**Comment by jarrpa** [comment]
@sdodson Anything to be done to get this going again?

**Comment by sdodson** [comment]
Needs lgtm, I forgot why I canceled lgtm. I think maybe because the bot was spinning on disabled fedora tests being red?
/lgtm
/test install
/test gcp
/test unit

**Comment by sdodson** [comment]
If those are all green we'll merge manually.

**Comment by vrutkovs** [comment]
PAPR tests are no longer run, merging manually

**Pull request merged**